frontend/fingertips-frontend/components/organisms/LineChartTable/index.tsx

```diff
@@ -1,19 +1,77 @@
 'use client';
 
 import { Table } from 'govuk-react';
-import { sortHealthDataByDate } from '@/lib/chartHelpers/formatChartValues';
 import { HealthDataForArea } from '@/generated-sources/ft-api-client';
+import styled from 'styled-components';
 
 interface TableProps {
-  data: HealthDataForArea[];
+  data: HealthDataForArea;
   headings: string[];
 }
 
+interface LineChartTableData {
+  period: number;
+  count: number;
+  value: number;
+  lower: number;
+  upper: number;
+}
+
+const StyledAlignCenterHeader = styled(Table.CellHeader)({
+  textAlign: 'center',
+  padding: '0.3125em',
+});
+
+const StyledAlignRightHeader = styled(Table.CellHeader)({
+  textAlign: 'right',
+  paddingTop: '0',
+});
+
+const StyledTable = styled(Table)({
+  marginBottom: '0.9375em',
+});
+
+const StyledTableCell = styled(Table.Cell)({
+  textAlign: 'left',
+});
+
+const mapToTableData = (areaData: HealthDataForArea): LineChartTableData[] =>
+  areaData.healthData.map((healthPoint) => ({
+    period: healthPoint.year,
+    count: healthPoint.count,
+    value: healthPoint.value,
+    lower: healthPoint.lowerCi,
+    upper: healthPoint.upperCi,
+  }));
+
 export function LineChartTable({ data, headings }: Readonly<TableProps>) {
-  data = sortHealthDataByDate(data);
+  const tableData = mapToTableData(data);
+  tableData.sort((a, b) => b.period - a.period);
```

Comment:
I think I would prefer if we created a new const of the sorted data. Rather than changing the original.

you can do this with something like this;

```
const dataSortedByPeriod = [...tableData].sort(Sort logic here);
```